dvc.lock

```diff
@@ -73,8 +73,8 @@ stages:
       size: 6238409
     - path: src/features/process_data.py
       hash: md5
-      md5: 015fb0faa169abec8d18aed0c9878fde
-      size: 1792
+      md5: 7bb019372acf7afe31f49e13100d3daa
+      size: 1893
```

Comment:
Would it be usefull to add this file to gitignore, it gets generated everytime you do dvc repro right?

Reply:
Ah you do want to track dvc.lock in Git so that DVC can determine the latest stage of the pipeline associated with the Git commit in every repo copy or branch.

Reply:
ah okay, makes sense